ext/standaloneusers/ang/crmResetPassword.js

```diff
@@ -58,7 +58,7 @@
         updateAngular('formSubmitted', true);
         updateAngular('pwnd', false);
         if (ctrl.newPassword.length < 8) {
-          alert(ts("Passwords under 8 characters are simply not secure. Ideally you should use a secure password generator."));
+          alert(ts("Passwords under 8 characters are not secure. Ideally you should use a secure password generator."));
```

Comment:
This is perhaps a stylistic choice, but I tend to side with: https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/